Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM ubuntu:18.04
+MAINTAINER Event Store Ltd <ops@eventstore.com>
+
+ENV ES_VERSION=5.0.8-1 \
```

Comment:
shouldn't this be an ARG instead?

Reply:
Adapted it from the current v5 Dockerfile from https://github.com/EventStore/eventstore-docker.

 Makes sense to be using ARG here! Updating it